src/index.html

```diff
@@ -11,10 +11,10 @@
     <div class="wrapp-list">
         <div class="header">
             <p>Today's To Do</p>
-            <i class="fas fa-sync"></i>
         </div>
         <div class="inputfield">
             <input type="text" placeholder="Add to your list ..." id="task">
+            <i class="fas fa-plus" id="btn-add"></i>
```

Comment:
- [ ] Great job so far you project work perfectly weel :clap:  whatever you should only display one button at once, the ` delete` button can only be displayed after making a click on the ` edit` button

Your:
![09](https://user-images.githubusercontent.com/57562869/149524730-1326ab9e-76a1-4a5f-89ab-45f0e2c956c3.png)


Original: 
![222](https://user-images.githubusercontent.com/57562869/149311739-24aa58a5-d3e2-40bc-9eea-7b7847b9d1d4.png)

Reply:
thanks for your review, I just want to know if this is 100 % require ?
just because the project works as respected.